tests/core_test.go

```diff
@@ -26,11 +26,24 @@ var _ = Describe("Core API endpoint tests", func() {
 
 	Context("Location endpoint", func() {
 
-		It("Should list all available locations", func() {
+		It("Should list all available locations, and get the first entry by ID and code", func() {
```

Comment:
Better have this at three separate tests imho, `It("Should list all available locations", ...)` and `It("Should get the first entry by ID", ...)` and `It("Should get the first entry by code", ...)`